_gabarits-jsonld/README.md

```diff
@@ -66,6 +72,15 @@ Saisissez les informations identifiant et décrivant la compagnie, le groupe ou
 ### [_location_](https://schema.org/location)
 Saisissez les informations identifiant et décrivant l'endroit où est présenté l'événement. L'entité décrite sous la propriété _location_ peut être de @type [Place](https://schema.org/Place) (un lieu physique) ou de @type [VirtualLocation](https://schema.org/VirtualLocation) (un lieu virtuel).
 
+### [_sameAs_](https://schema.org/sameAs)
+Saisissez les URIs d'_identifiants passerelles_ permettant d'identifier sans ambiguité les _entitées imbriquées_ dans l'entité de type _Event_. 
```

Comment:
I agree with you. This is a "big win" if we can get people to add sameAs URIs for People, Places, Organizations, Works.  In my opinion this is more significant than generating URIs for events.

Reply:
Please note that the latest commit integrates a footnote that explains what I mean by "identifiant" passerelle.